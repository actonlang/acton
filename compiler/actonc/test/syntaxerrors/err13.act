# Missing comma in list literal
def make_list():
    return [1 2 3]