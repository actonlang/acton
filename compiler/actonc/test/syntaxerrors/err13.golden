Building file test/syntaxerrors/err13.act using temporary scratch directory
[error Parse error]: unexpected '2'
                     expecting "for", ']', call arguments, slice/index expression, comma, if clause, or operator

     +--> test/syntaxerrors/err13.act@3:15-3:16
     |
   3 |     return [1 2 3]
     :               ^ 
     :               `- unexpected '2'
     :                  expecting "for", ']', call arguments, slice/index expression, comma, if clause, or operator
     :                  
-----+
